idc_index/idc_index_data/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""
+Copyright (c) 2024 Imaging Data Commons. All rights reserved.
+
+idc-index-data: ImagingDataCommons index to query and download data.
+"""
+
+
+from __future__ import annotations
+
+__version__ = "17"
```

Comment:
This sub-package illustrates what the standalone `idc-index-data` will look like.

Reply:
```suggestion
__version__ = "17.0"
```

I suggest we start versioning `idc-index-data` with `17.0` with `17` being the version of the `bigquery-public-data.idc_current.dicom_all_view` table, and `0` being the minor version we can use for maintenance fixes specific to the `idc-index-data` project.

Reply:
Yes, this makes a lot of sense and is a great idea. I would even do `17.0.0`. IDC data releases are coming out at the 2-3 months frequency, and it makes a lot of sense to expect that we may have updates of the index of varying complexity (e.g., adding new tables as opposed to fixing typos or including additional columns in the existing index).